### ISSUE TITLE
manage.py migrate command raises the error `relation "cache" does not exist`, which requires manage.py createcachetable command exec

### ISSUE THREAD

**Issue by sebgrebe** [rtcharity/eahub.org#1189]
### Current state
* When running ```docker-compose up web``` and then going on localhost:8000, getting error

```
relation "cache" does not exist
LINE 1: SELECT "cache_key", "value", "expires" FROM "cache" WHERE "c...
```

* Probably the case because no db present
* We used to be able to run without having local db (it would build one from scratch)

### Desired state
* Enable running project without having local db

**Comment by viktor-yunenko** [comment]
does `python manage.py createcachetable` help?

**Comment by sebgrebe** [comment]
@victor-yunenko, yes, thanks! Can we add this to the dockerfile such that if no local db is present, it runs it? Alternatively, we could put it in the README. Newcomers to the project will stumble on this

**Comment by viktor-yunenko** [comment]
We can add a django RunPython migration

**Comment by viktor-yunenko** [comment]
But we need to remember to add RunPython.noop as well

**Comment by sebgrebe** [comment]
Would you be happy to create a PR for this? @victor-yunenko
